src/components/pagination/_pagination.scss

```diff
@@ -189,3 +189,195 @@ $css--helpers: true;
     margin-bottom: 0;
   }
 }
+
+@mixin pagination--x {
+  .#{$prefix}--data-table-v2-container + .#{$prefix}--pagination {
+    border-top: 0;
+  }
+
+  .#{$prefix}--pagination {
+    @include reset;
+    @include font-family;
+    width: 100%;
+    background-color: $ui-01;
+    display: flex;
+    align-items: center;
+    border: 1px solid $ui-03;
+    height: rem(48px);
+
+    .#{$prefix}--form-item {
+      flex: auto;
+    }
+  }
+
+  .#{$prefix}--pagination__left {
+    display: flex;
+    height: 100%;
+    align-items: center;
+  }
+
+  .#{$prefix}--pagination__right {
+    display: flex;
+    height: 100%;
+    align-items: center;
+    margin-left: auto;
+  }
+
+  .#{$prefix}--pagination__text {
+    @include typescale('zeta');
+    color: $text-02;
+    display: none;
+    padding-right: $spacing-2xs;
+    padding-top: 1px;
```

Comment:
Was just wondering if it would be better to make this `rem(1px)`? Or the spacing token for `1px`? `$spacing-4xs`.